validator/lib/validatePnts.js

```diff
@@ -181,6 +183,16 @@ function validatePnts(content) {
         return 'Feature table property BATCH_LENGTH must be less than or equal to POINTS_LENGTH.';
     }
 
+    if (defined(featureTableJson.BATCH_ID)) {
+        var featureTable = new Cesium3DTileFeatureTable(featureTableJson, featureTableBinary);
+        var batchIds = featureTable.getPropertyArray('BATCH_ID', ComponentDatatype.UNSIGNED_SHORT, 1);
+        for (var i = 0; i < batchIds.length; i++) {
+             if (featureTableJson.BATCH_LENGTH <= batchIds[i]) {
```

Comment:
Reversing the condition is slightly clearer:

`if (batchIds[i] >= featureTableJson.BATCH_LENGTH) {`

Also tweak the description to match.